lib/recurly/bank-account.js

```diff
@@ -0,0 +1,181 @@
+
+/*!
+ * Module dependencies.
+ */
+
+var bind = require('bind');
+var each = require('each');
+var type = require('type');
+var index = require('indexof');
+var debug = require('debug')('recurly:bankAccount');
+var dom = require('../util/dom');
+var errors = require('../errors');
+
+/**
+ * expose
+ */
+
+module.exports = function bankAccount () {
+  return {
+    token: bind(this, token),
+    bankInfo: bind(this, bankInfo)
+  };
+};
```

Comment:
I'm not a huge fan of `Recurly.bankAccount().token` as the calling pattern. It would be more efficient to set up the binding once instead of on every call, and `bankAccount` should really be a module-like namespace instead of a callable function.

Can we export the unbound functions here, and then bind them in the main `Recurly` initializer?